packages/cli/src/fix.ts

```diff
@@ -44,7 +44,13 @@ async function fixPackage(pkg: Package): Promise<() => Promise<boolean>> {
     }
   }
 
-  if (!pkg.isTypeModule()) {
+  if (pkg.isTypeModule()) {
+    keys(fields).forEach((field) => {
+      if (fields[field]) {
+        delete (pkg.json as any)[field];
```

Comment:
Note this is done on `pkg.json` instead of the entrypoints since the entrypoint `package.json`s(besides the package's one itself) are removed automatically already elsewhere